content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,609 @@
+    +  uid: 0e735662-8768-4dd7-ae7f-7f93674d08b8
+    spec:
+      dnsNames:
+      - example.com
+    ```
+
+## 2 - Defaulting required fields
+
+> ⚠️ This section requires cert-manager v1.14.X to work properly out of the box.
+> If using an older version of cert-manager, see the [Appendix](#cert-manager-version-requirement) section for a full explanation.
+>
+
+Now we can set a Kyverno `ClusterPolicy` to apply default values to any of the `Certificate` fields.
+This includes the *required* fields. 
+In our example `ClusterPolicy` we will do two things:
+
+- Apply a default `secretName` that is the name of the Certificate object suffixed with "-cert".
+- Set the relevant `issuerRef` fields to default to using our `ClusterIssuer` named, "our-corp-issuer".
+
+> ℹ️ Note how the second rule is tackling the third of our [3 uses cases](#use-cases) set out above.
+
+1. Here is the `ClusterPolicy` resource to set both fields with defaults:
+
+    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/cpol-mutate-certificate-required.yaml
+    ```
+    🔗 <a href="cpol-mutate-certificate-required.yaml">`cpol-mutate-certificate-required.yaml`</a>
+
+1. Apply this policy:
+
+    ```shell
+    kubectl apply -f cpol-mutate-certificate-required.yaml
+    ```
+
+    You should now see two `ClusterPolicy` resources if you run:
+
+    ```shell
+    kubectl get cpol
+    ```
+
+    Example:
+
+    ```shell
+    NAME                            ADMISSION   BACKGROUND   VALIDATE ACTION   READY   AGE    MESSAGE
+    0-mutate-certificate-defaults   true        true         Audit             True    3m6s   Ready
+    1-mutate-certificate-required   true        true         Audit             True    4s     Ready
+    ```
+
+1. Patch the previous policy we applied, to disable schema validation after mutation:
+
+    ```shell
+    kubectl patch cpol 0-mutate-certificate-defaults --type=merge -p '{"spec":{"schemaValidation": false}}'
+    ```
+
+    > ⚠️ Read about the schema validation setting before using this elsewhere. 
+    > We have disabled it here because we have two policies both affecting `Certificates`. Cert-manager will validate the resolved resource with the `validatingwebhookconfigurations.admissionregistration.k8s.io`, after all mutations are applied.
+    > You could avoid this completely by using one policy with all required mutation rules.
+    >
+
+1. Take a look at another new `Certificate` we have for validating that all our policies will now take effect. This time called "test-minimal":
+
+    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/cert-test-minimal.yaml
+    ```
+    🔗 <a href="cert-test-minimal.yaml">`cert-test-minimal.yaml`</a>
+
+1. Dry run apply and `diff` to validate all our defaults have applied to this minimal `Certificate`:
+
+    ```shell
+    kubectl apply -f cert-test-minimal.yaml --dry-run=server -o yaml | diff -uZ cert-test-minimal.yaml -
+    ```
+
+    An example succesful output:
```

Comment:
`The output of this command should result in something similar to:`

I didn't like going from command to output without something. That is my new suggestion, removing the `success` part. I agree in a tutorial, it would be expected that all commands do indeed work.

Reply:
I can live with your proposal. Please use it consistently throughout.